packages/scripts/src/generate-master-list/generator.ts

```diff
@@ -229,19 +231,21 @@ const generateAirtableMasterList = async (adapterList: TableText) => {
 
     const version = unwrapCode(adapter[1])
     const type = unwrapCode(adapter[2])
-    const defaultApiUrl = unwrapCode(adapter[3])
-    const dependencies = unwrapCode(adapter[4])
-    const envVars = unwrapCode(adapter[5])
-    const endpoints = unwrapCode(adapter[6])
-    const defEndpoint = unwrapCode(adapter[7])
-    const batchableEndpoints = unwrapCode(adapter[8])
-    const isWSSupported = unwrapCode(adapter[9]) === '✅'
-    const hasUnitTests = unwrapCode(adapter[10]).includes('✅')
-    const hasIntegrationTests = unwrapCode(adapter[11]).includes('✅')
-    const hasE2ETests = unwrapCode(adapter[12]).includes('✅')
+    const frameworkVersion = unwrapCode(adapter[3])
```

Comment:
Based on the list of general fields below, shouldn't the indexes for `frameworkVersion` and `type` switched?